frontends/web/src/api/account.ts

```diff
@@ -324,3 +324,29 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
 export const testRegister = (pin: string): Promise<null> => {
   return apiPost('test/register', { pin });
 };
+
+export type TSignMessage = {
```

Comment:
Try changing the type to sth like this

https://github.com/digitalbitbox/bitbox-wallet-app/blob/c4375864155bffc353c11749e116d3adf2a0f9db/frontends/web/src/api/account.ts#L177

which enforces checking the success flag before accessing the fields only available on success.

same for the type below.